include/crypto/hpke.h

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright 2019-2021 Stephen Farrell. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
```

Comment:
This file is supposed to be public, isn't it? Then it should be in `include/openssl`, not `include/crypto`.
Also, we should discuss licensing...

Reply:
fixed in 32f0f8a64b10ff8c503b04fe71cbe47a7aabb968